### PR TITLE
Update release steps

### PR DESCRIPTION
**Pull request by fortmarek**
When doing the release of `0.69.0`, we have noticed the guide for releasing a stable minor had a few inconsistencies.

### CODE REVIEW

File: website/contributing/release-stable-minor.md [hunk 1]
```diff
@@ -29,14 +27,26 @@ title: Release Stable Minor
 
 0.66 stable is out!
```
Comment (Simek) [inline website/contributing/release-stable-minor.md:28]: Probably we should change the version in the whole article (currently we have the mix content mentioning 0.66 and 0.69) or just write the sentences in version-agnostic way (X.Y.Z).

Comment (fortmarek) [reply website/contributing/release-stable-minor.md:28]: I think it might be better to use a specific version, let me change it

Comment (kelset) [reply website/contributing/release-stable-minor.md:28]: maybe just use a `0.M.P` type of thing?

Comment (kelset) [reply website/contributing/release-stable-minor.md:28]: tbh it's fine either way 👍

**Review by kelset — COMMENTED** [review]
LGTM 👍

File: website/contributing/release-stable-minor.md (hunk 1 above)
Comment (fortmarek) [reply website/contributing/release-stable-minor.md:28]: If we want to use `0.M.P`, the better thing to do here would be to change it everywhere in the release steps. I don't mind doing it but maybe should be a different PR?

**Review by Simek — APPROVED** [review]
Thank you for the updates, LGTM! 👍

**Pull request merged**